BinaryEncoding.md

```diff
@@ -240,6 +239,42 @@ This section must be preceded by a [Functions](#functions-section) section.
 | count | `varuint32` | count of entries to follow |
 | entries | `uint16*` | repeated indexes into the function table |
 
+### Function Names section
+
+ID: `function_names`
+
+| Field | Type | Description |
+| ----- |  ----- | ----- |
+| names | `utf8-C-string*` | sequence of null-terminated utf8-encoded strings |
```

Comment:
I'm well aware :)

I guess my point was: the annoyance of internal null-terminators is not specific to the web (as you said it works just fine as a string); and the whole point of these strings is to inject into bigger strings.  I guess it just boils down to me being lazy: I'd rather deal with utf8 char* C-strings than {length, char*} pairs :)  So I guess I'm fine dropping this restriction too, if others want to allow.